router/mount.go

```diff
@@ -40,7 +40,7 @@ func (m *ServerlessRouterMount) WrappedHandler() http.HandlerFunc {
 
 		for _, value := range m.BinaryMediaTypes {
 			mediaType, _, err := mime.ParseMediaType(contentType)
```

Comment:
Haven't looked at the context of this code so feel free to ignore this comment. Are you parsing the contentType header value on each cycle of the loop? Might as well do it once outside.

Reply:
I moved it outside of the loop. Thanks!